apps/game_backend/lib/game_backend/configuration/version.ex

```diff
@@ -0,0 +1,21 @@
+defmodule GameBackend.Configuration.Version do
+  @moduledoc """
+  Version schema.
+  """
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  schema "versions" do
+    field(:name, :string)
+    field(:current, :boolean)
+
```

Comment:
Should we have the `has_one` relations?

Reply:
Thanks for the heads up, it let me do the following refactor 

- [Add has_one and has_many relationship to version schema](https://github.com/lambdaclass/mirra_backend/pull/806/commits/d6597454eaec8dae75711b144e9456f5450c03db)
- [Use the has one and many relationships to get the data from a current version](https://github.com/lambdaclass/mirra_backend/pull/806/commits/10b52bbe5c3d159ff8ddb0d6a4876eb116a55cf7)